standards/choosing_your_mug.md

```diff
@@ -0,0 +1,11 @@
+# Choosing your Mug.
+
+This is very important - as your mug can ofter give people the first impression of you. 
+You want your mug to refelct your personality. 
+If you dont have one - please use a blank mug. 
+
+## Washing of mugs.
+
+- Having a dirty mug is **not** acceptable
+- Asking some one to clean your mug is **not** acceptable 
+- However offering to clean a friends mug is *highly* acceptable
```

Comment:
I would also suggest that mugs should not be used for anything other than tea, coffee or water due to the smell of other liquids.